modules/tinymce/src/themes/silver/main/ts/ContextToolbar.ts

```diff
@@ -79,7 +95,7 @@ const getAnchorLayout = (position: InlineContent.ContextPosition, isTouch: boole
   } else {
     return {
       bubble: Bubble.nu(0, bubbleSize, bubbleAlignments),
-      layouts: isTouch ? mobileAnchorSpecLayouts : desktopAnchorSpecLayouts,
+      layouts: isTouch ? mobileAnchorSpecLayouts : getDesktopAnchorSpecLayouts(contextbar),
```

Comment:
The `contextbar` is required because we detect what the current layout is on it (for keeping pinned layouts the same). Unfortunately, that information isn't available in the onLtr function ... it just gets the node anchor, not the element being shown.